pipeline/tasks/gapic_tasks.py

```diff
@@ -134,7 +134,7 @@ def execute(self, language, api_name, final_repo_dir):
         # Such APIs don't need packman.
         if ((language == 'ruby' or language == 'nodejs') and
                 task_utils.is_output_gcloud(final_repo_dir)):
-            return
+            return final_repo_dir
```

Comment:
This logic doesn't make sense to me -- if `package_dir == final_repo_dir` always, why do we need two variables?